### ISSUE TITLE
The settings for 3 runs in the 3BPA and AcAc datasets

### ISSUE THREAD

**Issue by ndlongvn** [CLaSLoVe/MMEA#2]
Hi. In the 3BPA and AcAc dataset training settings, the paper notes that the results are got from 3 different runs. Are different runs using different commands or different seeds, or different settings? Can you please clarify it? Thank you.

**Comment by CLaSLoVe** [comment]
Hi ndlongvn. We followed the official ELoRA implementation and kept the same experimental protocol across runs. We also kept the same seed to maintain a consistent data split.

**Comment by ndlongvn** [comment]
> Hi ndlongvn. We followed the official ELoRA implementation and kept the same experimental protocol across runs. We also kept the same seed to maintain a consistent data split.

So you mean you use the same command and run it 3 times, right?
